qemu/tests/cfg/rng_driver_load.cfg

```diff
@@ -9,10 +9,12 @@
             type = driver_load
             driver_id_pattern = virtio_rng
             driver_id_cmd = lsmod | grep virtio_rng
-            driver_unload_cmd = modprobe -r virtio_rng
-            driver_load_cmd = modprobe virtio_rng
+            driver_unload_cmd = "modprobe -r virtio_rng"
+            driver_load_cmd = "modprobe virtio_rng"
+            stop_service_cmd = "service rngd stop"
+            start_service_cmd = "service rngd restart"
             repeats = 10
         - driver_in_use:
             type = rng_driver_negative
-            driver_unload_cmd = modprobe -r virtio_rng
-            read_rng_cmd  = "dd if=/dev/random of=/dev/null & echo $!"
+            driver_unload_cmd = "modprobe -r virtio_rng"
```

Comment:
I prefer to move `driver_unload_cmd` outside since both cases 'load_in_loop' and 'driver_in_use' use the same value.